src/test/java/com/spaceclub/event/controller/EventControllerTest.java

```diff
@@ -324,4 +331,35 @@ class EventControllerTest {
                 ));
     }
 
+    @Test
+    @WithMockUser
+    void 개별_행사_북마크_상태를_변경에_성공한다() throws Exception {
```

Comment:
메소드 이름을 다음과 같이 변경하는 것은 어떠신가요?
```suggestion
    void 개별_행사_북마크_상태를_변경하는데_성공한다() throws Exception {
```
```suggestion
    void 개별_행사_북마크_상태_변경에_성공한다() throws Exception {
```

Reply:
상태 변경에 성공한다로 변경할게요